packages/wonder-blocks-button/src/components/__tests__/button.test.js

```diff
@@ -785,6 +786,56 @@ describe("Button", () => {
         });
     });
 
+    describe("button focus", () => {
+        test("can be focused", async () => {
+            // Arrange
+            render(<Button testId={"button-focus-test"}>Label</Button>);
+
+            // Act
+            const button = screen.getByTestId("button-focus-test");
+            button.focus();
+
+            // Assert
+            expect(button).toHaveFocus();
+        });
+
+        test("can be focused when disabled", async () => {
+            // Arrange
+            render(
+                <Button disabled={true} testId={"button-focus-test"}>
+                    Label
+                </Button>,
+            );
+
+            // Act
+            const button = screen.getByTestId("button-focus-test");
+            button.focus();
+
+            // Assert
+            expect(button).toHaveFocus();
+        });
+
+        test("can be focused when disabled", async () => {
```

Comment:
I think we need to update the names of these tests to make it clear we're testing primary/tertiary/etc.